oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/IndexUpdate.java

```diff
@@ -541,6 +541,12 @@ public void onMissingIndex(String type, NodeBuilder definition, String indexPath
                 return;
             }
 
+            PropertyState providerPropertyState =  definition.getProperty("ignoreMissingProvider");
```

Comment:
I see an existing option `failOnMissingIndexProvider`. Should we reuse this instead?

Reply:
That one is a global configuration for all indexes. We need something specific to elastic search as we don't want other indexes to be affected. 
However, I am thinking of removing this config, and just let indexing succeed as this is not making much difference other than avoiding setting the reindexCount property.